### PR TITLE
change Error actions payload to HttpErrorResponse.error object

### PR DESCRIPTION
**Pull request by soundTricker**
for #60 
Note: I know this PR break a Backward Compatibility....

### CODE REVIEW

**Comment by soundTricker** [comment]
@fazpu Sorry, I'm late. I've fixed.

File: src/forms/states/generate-http-effects.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ function getEffectDefinition(actionClassNameBase: string, name: string, simpleNa
     `this.${name.toLowerCase()}Service.${simpleName}(${startActionPayloadDefinition})\n`, 2);
   res += indent(`.pipe(\n`, 3);
   res += indent(`map(result => new actions.Success(result)),\n`, 4);
-  res += indent(`catchError((error: HttpErrorResponse) => of(new actions.Error(error.message))),\n`, 4);
+  res += indent(`catchError((error: HttpErrorResponse) => of(new actions.Error(error.error))),\n`, 4);
```
Comment (fazpu) [inline src/forms/states/generate-http-effects.ts]: please, make it just `error` instead of `error.error`

**Comment by fazpu** [comment]
@soundTricker Thank you very much! Please work in the comment above, re-generate the demo files and rebase.

**Comment by soundTricker** [comment]
@fazpu sorry, I've fixed.

**Comment by fazpu** [comment]
Lets rebase and merge. Thanks!

**Comment by soundTricker** [comment]
@fazpu  Soooooooooo sorry...... I've rebase & merged .

**Pull request merged**